mysql-test/suite/s3/hw_obs.test

```diff
@@ -0,0 +1,23 @@
+--source include/have_s3.inc
```

Comment:
is this test still needed? Does it test some huaweicloud specific functionality?

Reply:
Hello, vuvova, I just simulated amazon test case.

Reply:
The s3/amazon.test is there to test the s3_protocol_version="Original". From the version in hw_obs test this protocol isn't supported in huaweicloud.

With the other changes you have made, huawei obs can be checked with the appropriate set of ENV variables and this explictit test isn't required.